tests/page-objects/SurveyPage.ts

```diff
@@ -2,52 +2,121 @@ import { Selector, t } from 'testcafe';
 import { clickSelector } from './formHelper';
 
 export class SurveyPage {
+
     private readonly pageWrapper: Selector = Selector('.survey');
-    private readonly answer1Input: Selector = Selector('[name="answers[0].answer"]');
-    private readonly answer2Input: Selector = Selector('[name="answers[1].answer"]');
-    private readonly answer3Input: Selector = Selector('[name="answers[2].answer"]');
-    private readonly doneSkipButton = '.survey button';
+    private readonly page1: Selector = Selector('#survey-part-1');
+    private readonly submittingAs: Selector = Selector('[name="submittingAs"]');
+    private readonly independentResearcher: Selector = Selector('[name="independentResearcher"]');
+    private readonly independentResearcherYear: Selector = Selector('[name="independentResearcherYear"]');
+
+    private readonly page2: Selector = Selector('#survey-part-2');
+    private readonly genderIdentity: Selector = Selector('[name="genderIdentity"]');
+    private readonly genderSelfDescribe: Selector = Selector('[name="genderSelfDescribe"]');
+    private readonly countryOfResidence: Selector = Selector('[name="countryOfResidence"]');
+    private readonly secondCountryOfResidenceToggle: Selector = Selector('#secondCountryOfResidenceToggle');
+    private readonly secondCountryOfResidence: Selector = Selector('[name="secondCountryOfResidence"]');
+    private readonly countryIndentifyAs: Selector = Selector('[name="countryIndentifyAs"]');
+    private readonly raceOrEthnicity: Selector = Selector('[name="raceOrEthnicity"]');
+
+    private readonly doneSkipButton = 'button[type="submit"]';
 
     public async assertOnPage(): Promise<void> {
+        // Container is present
         await t.expect(this.pageWrapper.exists).ok();
-        await t.expect(this.answer1Input.exists).ok();
-        await t.expect(this.answer2Input.exists).ok();
-        await t.expect(this.answer3Input.exists).ok();
+    }
+
+    private async assertOnPage1(): Promise<void> {
+        // Container is present
+        await t.expect(this.page1.exists).ok();
+
+        // Visible fields are present
+        await t.expect(this.submittingAs.exists).ok();
+        await t.expect(this.independentResearcher.exists).ok();
+
+        // Hidden fields are not visible
+        await t.expect(this.independentResearcherYear.exists).notOk();        
```

Comment:
The function name `assertOnPage1` doesn't specify that the form is clear of any values. We might get caught out with this assertion if we were to add extended survey tests later for navigating forward then back to page 1. I think the unit tests cover the conditionality of the `independentResearcherYear` without the need for an additional check here.